ui/raidboss/data/04-sb/raid/o2s.js

```diff
@@ -52,6 +59,45 @@ export default {
       condition: Conditions.targetIsYou(),
       run: (data) => data.levitating = false,
     },
+    {
+      id: 'O2S Evilsphere',
+      netRegex: NetRegexes.startsUsing({ id: '2371', source: 'Catastrophe' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '2371', source: 'Katastroph' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '2371', source: 'Catastrophe' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '2371', source: 'カタストロフィー' }),
+      netRegexCn: NetRegexes.startsUsing({ id: '2371', source: '灾变者' }),
+      netRegexKo: NetRegexes.startsUsing({ id: '2371', source: '카타스트로피' }),
+      condition: Conditions.caresAboutMagical(),
+      response: Responses.tankBusterSwap(),
+    },
+    {
+      id: 'O2S 100Gs',
+      netRegex: NetRegexes.headMarker({ id: '0069' }),
+      preRun: (data, matches) => {
+        data.blueCircle = data.blueCircle || [];
+        data.blueCircle.push(matches.target);
+      },
+      delaySeconds: 0.3,
+      response: (data, _matches, output) => {
+        // cactbot-builtin-response
+        output.responseOutputStrings = {
+          onYou: {
+            en: '100Gs on YOU',
+          },
+          beIn: {
+            en: 'Be in 100Gs Circle',
```

Comment:
```suggestion
            en: 'Be in 100Gs Circle',
            ko: '중력 100 원 안에 있기',
```

Reply:
I wonder if these should be like `Stack On X, Y (Go Down)`?

Reply:
Yeahhhh I thought about that, but it's more complicated to write.  This isn't like "two stack markers that need to spread" it's more just "everybody clump up together" and so it's less important who it's on (in my mind)?